src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java

```diff
@@ -0,0 +1,66 @@
+package cms.gov.madie.measure.services;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.springframework.stereotype.Service;
+
+import cms.gov.madie.measure.repositories.MeasureRepository;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import gov.cms.madie.models.measure.Measure;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+public class MeasureTransferService {
+  private final MeasureRepository measureRepository;
+
+  public List<Measure> findByMeasureSetId(String measureSetId) {
+    List<Measure> measures = measureRepository.findAllByMeasureSetId(measureSetId);
+    return measures;
+  }
```

Comment:
This seems more generic than just measure transfer. Consider moving to MeasureService.

Reply:
@jkotanchik-SB Ok thanks will do.

Reply:
@jkotanchik-SB This is done too.